### PR TITLE
[ONNX] Default runtime type checking to raising errors

### PR DESCRIPTION
**Pull request by justinchuby**
Default runtime type checking to raise by changing the default value to  `GLOBALS.runtime_type_check_state` into ERRORS

### CODE REVIEW

File: torch/onnx/_internal/_beartype.py [hunk 1]
```diff
@@ -45,18 +45,12 @@ def _create_beartype_decorator(
...
-                "but the beartype library is not installed. "
-                "Install beartype with `pip install beartype` to enable runtime type checking."
-            )
         return _no_op_decorator
 
     assert isinstance(_beartype_lib, ModuleType)
 
     if runtime_check_state == _exporter_states.RuntimeTypeCheckState.ERRORS:
-        # Enable runtime type checking which errors on any type hint violation.
+        # Enable runtime type checking with errors on any type hint violation.
```
Comment (BowenBao) [inline torch/onnx/_internal/_beartype.py:53]: verb to noun for errors 😆

**Review by BowenBao — APPROVED** [review]

File: torch/onnx/_internal/_beartype.py (hunk 1 above)
Comment (justinchuby) [reply torch/onnx/_internal/_beartype.py:53]: Ha! I should keep the original. For some reason my brain forgot that error can be a verb

**Pull request closed without merging**